src/Microsoft.OData.Core/UrlValidation/ValidationEngine/ODataUrlValidator.cs

```diff
@@ -190,12 +190,7 @@ private Type[] GetInterfaces(Type type)
             Type[] interfaces;
             if(!implementedInterfaces.TryGetValue(type, out interfaces))
             {
-
-#if NETSTANDARD1_1
                 interfaces = type.GetInterfaces().ToArray();

```

Comment:
Remove this and keep `interfaces = type.GetInterfaces();`
```suggestion
                interfaces = type.GetInterfaces();
```

Reply:
# Resolved.